src/frontend/utils/dato.ts

```diff
@@ -0,0 +1,15 @@
+import { format, isValid, startOfDay, startOfToday } from 'date-fns';
+
+import { Datoformat } from './formatter';
+
+export const tidenesMorgen = () => startOfDay(new Date(1000, 0));
+
+export const tidenesEnde = () => startOfDay(new Date(3000, 0));
+
+export const dagensDato = () => startOfToday();
+
+export const formatterDateTilIsoString = (dato?: Date): string =>
+    dato && isValid(dato) ? format(dato, Datoformat.ISO_DAG) : '';
```

Comment:
Liker egentlig ikke denne helt, for tom streng er ikke et bra alternativ. Har skrevet en annen kommentar på dette, hvor jeg lurer på hvordan man kan håndtere steder hvor backend krever string, men dato er ugyldig